packages/fastify-podlet-server/README.md

```diff
@@ -570,3 +570,29 @@ export default {
 ```
 
 See the [Fastify docs](https://www.fastify.io/docs/latest/Reference/Validation-and-Serialization/) for more
+
+## Errors
+
+Error handler has been added to enable developers to control application errors.
```

Comment:
Is this rather intended to start with `An error handler` or `Error handling`?

Reply:
Whoops 👍